ledger/ledger-api-bench-tool/README.md

```diff
@@ -17,6 +17,53 @@ bazel build //ledger/ledger-api-bench-tool:ledger-api-bench-tool_deploy.jar
 java -jar bazel-bin/ledger/ledger-api-bench-tool/ledger-api-bench-tool_deploy.jar --help
 ```
 
+## Configuration
+
+### Observer Parties
+
+You can specify the number of observer parties to create in the .yaml config as follows:
+```yaml
+submission:
+  num_observers: <NUMBER-OF-OBSERVERS>
+```
+This will tell the BenchTool to create `NUMBER-OF-OBSERVERS` parties.  
+The names of these parties will follow the pattern of `Obs-<INDEX>` where `0 <= INDEX < NUMBER-OF-OBSERVERS`
+and each party will have its own unique probability of `1 / 10^INDEX` for being selected as an observer of a contract.
+
+For example, when creating four observer parties they will be named `Obs-0`, `Obs-1`, `Obs-2`, `Obs-3`
+and their respective probabilities will be 100%, 10%, 1% and 0.1%.
+
+### Ledger Offset
+
+You can use any valid ledger offset.
+Additionally, you can use two special values `"ledger-begin"` and `"ledger-end"`
+
+### Consuming and Nonconsuming Exercises
+
+You can specify consuming and nonconsuming exercises to submit as below.
+```yaml
+submssion:
+  nonconsuming_exercises:
+    probability: 2.3
+    payload_size_bytes: 1000
+  consuming_exercises:
+    probability: 0.4
+    payload_size_bytes: 200
+```
+For nonconsuming exercises `probability` can be any positive number. 
+For example probability of `2.3` means that, for each create contract command,
+there will be at least two nonconsuming exercises submitted and 30% chance of a third one.
+
+For consuming exercises `probablity` must in range `[0.0, 1.0]`.
+
+### Archives
+NOTE: An vanilla archive command can be generated only if no other consuming exercise has been generated.  
+For example, given archive chance of `0.3` and `consuming_exercises.probability` of `0.4`, the chance of submitting 
+a vanilla archive command is 
```

Comment:
What is a vanilla archive? A command that just archives a contract as opposed to a command which has both create and archive?

Reply:
An `Archive` choice vs. user defined consuming choice

Reply:
Ok. Can you highlight it so it's clear that `Archive` there is the choice name? (just making it start with Capital will do)

Reply:
Ok, done